README.md

```diff
@@ -1,150 +1,140 @@
-
-| Name                                        | Hintergrund                                         |
-| ------------------------------------------- | --------------------------------------------------- |
-| **Baufinanzierungsereignisse lesen**        | Grundsätzlich zum Auslesen von Ereignissen benötigt |
-| **Baufinanzierung-Echtgeschäft bearbeiten** | sonst sind nur Ereignisse zu Testvorgängen lesbar   |
-| **Darf Partner-Daten lesen**                | sonst sind werden nur `PartnerIds` ausgegeben       |
-| **Dokumente lesen**                         | sonst werden keine Dokumente ausgegeben             |
-
-Schlägt die Authentifizierung fehl, erhält der Aufrufer eine HTTP Response mit Statuscode **401 UNAUTHORIZED**.
-
-Hat der Client nicht die benötigte Berechtigung um die Resource abzurufen, erhält der Aufrufer eine HTTP Response mit Statuscode **403 FORBIDDEN**.
-
-## TraceId zur Nachverfolgbarkeit von Requests
-
-Für jeden Request soll eine eindeutige ID generiert werden, die den Request im EUROPACE 2 System nachverfolgbar macht und so bei etwaigen Problemen oder Fehlern die systemübergreifende Analyse erleichtert.  
-Die Übermittlung der X-TraceId erfolgt über einen HTTP-Header. Dieser Header ist optional,
-wenn er nicht gesetzt ist, wird eine ID vom System generiert.
-
-| Request Header Name | Beschreibung                    | Beispiel    |
-|---------------------|---------------------------------|-------------|
-| X-TraceId           | eindeutige Id für jeden Request | sys12345678 |
-
-## Content-Type
-
-Die Schnittstelle akzeptiert Daten mit Content-Type `application/json`.  
-Entsprechend muss im Request der Content-Type Header gesetzt werden. Zusätzlich das Encoding, wenn es nicht UTF-8 ist.
-
-| Request Header Name |   Header Value   |
-|---------------------|------------------|
-| Content-Type        | application/json |
-
-## Fehlercodes
-
-### HTTP-Status Errors
-
-| Fehlercode | Nachricht       | weitere Attribute          | Erklärung                            |
-|------------|-----------------|----------------------------|--------------------------------------|
-| 401        | Unauthorized    | -                          | Authentifizierung ist fehlgeschlagen |
-
-### Weitere Fehler
-
-| Fehlercode | Nachricht                  | Erklärung                                                                                       |
-|------------|----------------------------|-------------------------------------------------------------------------------------------------|
-| 403        | Insufficient access rights | Es wird versucht auf eine Ressource zuzugreifen, die die Vertriebsorganisation nicht lesen darf |
-| 404        | Ressource not found | Es wird versucht auf eine nicht existierende Ressource zuzugreifen |
-| 503        | Service temporarily unavailable | Informationen zu Partner, Dokumenten oder dem Vorgang sind aktuell nicht verfügbar |
```

Comment:
Warum wird das hier einfach weggeworfen? 
Das sind durchaus relevante Informationen, die das Debuggen und Verstehen fördern.
Im Zweifel wenigstens nochmal einen Link zur Doku anbieten

Reply:
Fehlercodes sind Spezifikationen die in die YAML gehören im besten Fall mit Erklärung. Diese wurde hier https://europace.github.io/baufismart-ereignisse-api/gh-pages/index.html#overview verlinkt.